glusterd2/commands/snapshot/snapshot-create.go

```diff
@@ -37,6 +38,11 @@ import (
 	config "github.com/spf13/viper"
 )
 
+type txnData struct {
+	Req      api.SnapCreateReq
+	SnapTime time.Time
```

Comment:
is SnapTime is snapshot created Time?

Reply:
Yes

Reply:
I think then we can rename it to ```CreatedAt``` for better readability

Reply:
This sounds much better. Change the `Snapinfo` struct to this as well.

Reply:
done